sympy/physics/control/lti.py

```diff
@@ -2479,6 +2497,9 @@ def doit(self, cancel=False, expand=False, **hints):
     def _eval_rewrite_as_TransferFunction(self, num, den, sign, **kwargs):
         return self.doit()
 
+    def to_expr(self):
+        return self.doit().to_expr()
```

Comment:
Actually, do you need this defined at all? Doesn't TransferFunction do the same thing and this behavior is inherited?

Reply:
No Sir, It was giving an error when I've not defined it.

Reply:
to_expr() does not call doit() in TF:

https://github.com/sympy/sympy/blob/master/sympy/physics/control/lti.py#L1273

So this is fine, but you need a docstring.

Reply:
I've written the docstring in the recent commit.